src/generative_models/GenerativeModels.jl

```diff
@@ -1,6 +1,14 @@
 module GenerativeModels
 
 using CounterfactualExplanations
+using CUDA
+using Flux
+using Flux
+using Flux.Losses
```

Comment:
not required

Reply:
Flux.Losses you think is not required?

Reply:
Since you write `Flux.Losses.mse` fully out in `vae_params.jl`, I also think just importing Flux would be enough